core/templates/pages/exploration-editor-page/preview-tab/preview-tab.component.ts

```diff
@@ -152,8 +152,7 @@ export class PreviewTabComponent
           () => {
             this.loadPreviewState(initStateNameForPreview, []);
           });
-        this.playerCorrectnessFeedbackEnabledService.init(
-          returnDict.correctness_feedback_enabled);
+        this.playerCorrectnessFeedbackEnabledService.init(true);
```

Comment:
Why do we need this line (and the service in question)?

Reply:
this was being used in exploration editor to get correctness feedback state and update it, which we dont need anymore so removed the entire service.